Nos/Models/Event+CoreDataClass.swift

```diff
@@ -77,6 +77,8 @@ public class Event: NosManagedObject {
     
     static var replyNoteReferences = "kind = 1 AND ANY eventReferences.referencedEvent.identifier == %@ " +
         "AND author.muted = false"
+    public static var discoverKinds = [EventKind.text, EventKind.longFormContent]
```

Comment:
I think it makes sense to define this somewhere but unfortunately I don't know where. Suggestions are very much welcome.

Reply:
this works! I might have put it on `EventKind` but this makes just as much sense I think.